src/Model/GContact.php

```diff
@@ -924,7 +924,7 @@ private static function updateFromPublicContact($condition)
 		}
 
 		// These fields are having different names but the same content
-		$gcontact['server_url'] = $contact['baseurl'];
+		$gcontact['server_url'] = defaults($contact, 'baseurl', ''); // "baseurl" can be null, "server_url" not
```

Comment:
If `$contact['baseurl']` is guaranteed to exist at this point, please write `$contact['baseurl'] ?? ''` instead.

Reply:
I really, really, really have to work myself into these structures. I always have to look at examples to understand it. What is it checking? ```$contact['baseurl']``` can be null, but does exist at any time.

Reply:
`$var ?: $default` is `$var ? $var : $default`. However it doesn't work with non-existing variables/array keys.

`$var ?? $default` is `isset($var) ? $var : $default` so it doesn't complain for non-existing variables/array keys but it doesn't fall back to the default for falsy values.